src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -23,13 +23,13 @@
 import com.google.api.services.youtube.YouTube;
 import com.google.gson.Gson;
 import java.io.IOException;
-import java.lang.Exception.ServletException;
 import java.security.GeneralSecurityException;
 import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
```

Comment:
When I compile, there is the following message: 

import java.security.GeneralSecurityException;  --> cannot resolve 'ServletException'

Reply:
I do not get that error, and therefore I don't know if I can recreate this bug. We should definitely talk about resolving it as a group